addons/transitions/Transitions.gd

```diff
@@ -115,7 +130,8 @@ func _common_wait_for_fade(data:Array, fade_type, fade_seconds:float) -> void:
 	
 	if fade_type == FadeType.CrossFade or fade_type == FadeType.Instant:
 		var tween = Tween.new()
-		_root.add_child(tween)
+		_root.call_deferred("add_child", tween)
+        yield(tween, "ready")
```

Comment:
Sorry, I meant this line. I get a runtime error about mixing tabs and spaces. This line seems to use spaces. (Here's how it looks in my text editor with whitespace characters showing: notice the -> on other lines, but ... here

![image](https://user-images.githubusercontent.com/31222662/163857550-342d93b9-7d76-4ee5-95c8-6d59ec026bbe.png)

Reply:
I missed that one, just edited it.